Content.Shared/RetractableItemAction/RetractableItemActionSystem.cs

```diff
@@ -0,0 +1,102 @@
+using Content.Shared.Actions;
+using Content.Shared.Hands.EntitySystems;
+using Content.Shared.Interaction.Components;
+using Robust.Shared.Audio.Systems;
+using Robust.Shared.Containers;
+using Robust.Shared.Network;
+
+namespace Content.Shared.RetractableItemAction;
+
+/// <summary>
+/// System for handling retractable items, such as armblades.
+/// </summary>
+public partial class RetractableItemActionSystem : EntitySystem
```

Comment:
```suggestion
public sealed class RetractableItemActionSystem : EntitySystem
```
don't ignore the IDE warnings